recipe/meta.yaml

```diff
@@ -22,24 +22,26 @@ build:
 requirements:
   build:
     - {{ compiler('cxx') }}  # [not win]
-    - clang  # [win]
+    - clang                  # [win]
+    - clangxx                # [win]
   host:
     - python
     - pip
-    - numpy >=1.13
+    - numpy
```

Comment:
Numpy pinnings are currently at 1.16.

In either way, this had unintended consequences since it would previously have selected the very latest numpy version available at compile time.

conda-forge wishes to select the oldest supported version that meets the minimum requirements.

The "appropriate" syntax at the time would have been `numpy 1.13` with a comment to unpin when the conda-forge pinnings are updated.